hazelcast/src/main/java/com/hazelcast/spi/properties/GroupProperty.java

```diff
@@ -122,8 +123,9 @@
      * <p/>
      * By default only mandatory probes are being tracked
      */
-    public static final HazelcastProperty PERFORMANCE_METRICS_LEVEL
-            = new HazelcastProperty("hazelcast.performance.metric.level", ProbeLevel.MANDATORY.name());
+    public static final HazelcastProperty PERFORMANCE_METRICS_LEVEL =
+            new HazelcastProperty(Diagnostics.PREFIX + ".metric.level", ProbeLevel.MANDATORY.name())
```

Comment:
I'm going to move this property to the Diagnostics; like all other Diagnostics properties. So it will not be in the SPI any longer.